CMakeLists.txt

```diff
@@ -165,3 +165,22 @@ install(
     DESTINATION
         ${SHARE_INSTALL_PREFIX}/${APPLICATION_NAME}
 )
+
+install(
+    FILES
+        tools/pylintrc
+        tox.ini
+    DESTINATION
+        ${SHARE_INSTALL_PREFIX}/${APPLICATION_NAME}/tests
+)
+
+install(
+    FILES
+        tools/pki-lint
```

Comment:
If possible we should just move `pylintrc`, `tox.ini`, and `pki-lint` source into the `tests` so the source layout is closer to the actual installation layout.

Reply:
`pki-lint` script will go away when we migrate to setuptools
`tox.ini` needs to stay in the root dir
`pylintrc` contains set of rules rather than tests itself. So, I'm inclined to keep it inside `tools/` for now.